code/drasil-printers/Language/Drasil/TeX/Print.hs

```diff
@@ -56,7 +56,7 @@ buildStd sm (Document t a c) =
 lo :: LayoutObj -> PrintingInformation -> D
 lo (Header d t l)       _  = sec d (spec t) %% label (spec l)
 lo (HDiv _ con _)       sm = print sm con -- FIXME ignoring 2 arguments?
-lo (Paragraph contents) _  = toText $ spec contents
+lo (Paragraph contents) _  = toText (spec contents) %% newline
```

Comment:
I'm guessing it's the `%%` which adds the newline, not the `newline` itself. This is going to be confusing for others!